packages/templates/nextjs-marketing-demo/src/components/Footer/FooterDebugging.tsx

```diff
@@ -0,0 +1,27 @@
+'use client';
+
+import React from 'react';
+import reactPackage from 'react/package.json';
+import { useSearchParams } from 'next/navigation';
+import { version as sdkVersion } from '@contentful/experiences-sdk-react';
+import { Divider, Flex, Typography } from 'antd';
+import styles from './styles.module.css';
+
+const FooterDebugging: React.FC = () => {
+  const searchParams = useSearchParams();
+  const isDebugMode = searchParams.has('debug');
+
+  if (!isDebugMode) {
```

Comment:
wonder if we can check for the dev env var here instead of passing in a query param